aten/src/ATen/core/function_schema.h

```diff
@@ -320,8 +320,8 @@ struct FunctionSchema {
     alias_kind_ = v;
   }
 
-  void setNamespaceIfNotSet(const char* ns) {
-    name_.setNamespaceIfNotSet(ns);
+  bool setNamespaceIfNotSet(const char* ns) {
```

Comment:
nit: comment on when this returns true/false. Or even better, make it return an enum with two possible (and descriptive) values.